### PR TITLE
use jenkins pipeline script from calance workflows repo

### PR DESCRIPTION
**Pull request by 12345pradeep**

### CODE REVIEW

File: jenkins-pipeline-scripts/kubernetes-deploy-on-nutanix.groovy [hunk 1]
```diff
@@ -0,0 +1,65 @@
+def imageName = ''
```
Comment (agoyalcalance) [inline jenkins-pipeline-scripts/kubernetes-deploy-on-nutanix.groovy:1]: are we planning to use this groovy script for running the pipeline? I think this should sit in the GitOps repository that we intend to create later on to manage helm values. @ypmittal thoughts?

Comment (ypmittal) [reply jenkins-pipeline-scripts/kubernetes-deploy-on-nutanix.groovy:1]: My though was that all pipeline code should reside in one place. Other repository will host all the infra configs and app manifests. What do you think @agoyalcalance?

Comment (agoyalcalance) [reply jenkins-pipeline-scripts/kubernetes-deploy-on-nutanix.groovy:1]: I am okay with that... only thing is we will have to clone this repository as well to just select Jenkinsfile, which might be an overhead. not sure if that is avoidable.

**Comment by ypmittal** [comment]
closing this as we are storing the jenkinsfile script in abnother repo

**Review by ypmittal — APPROVED** [review]
LGTM

**Pull request merged**